lib/chat_api/conversations.ex

```diff
@@ -23,6 +23,26 @@ defmodule ChatApi.Conversations do
     |> Repo.all()
   end
 
+  @spec list_conversations_by_account_paginated(binary(), map()) :: Paginator.Page.t()
+  def list_conversations_by_account_paginated(account_id, filters \\ %{}) do
```

Comment:
I just realized that you had this comment in the issue 
> This new method should just be called something like ChatApi.Conversations.list_conversations_by_account(account_id, filters, pagination_options), and should only return, let's say, 50 records at a time

Should i change this method to that signature?